packages/trpc/server/routers/publicViewer/i18n.handler.ts

```diff
@@ -11,7 +11,7 @@ type I18nOptions = {
   input: I18nInputSchema;
 };
 
-export const i18nHandler = async ({ input }: I18nOptions) => {
+export const i18nHandler = async ({ ctx, input }: I18nOptions) => {
```

Comment:
Is ctx being used here?

Reply:
It is not, I removed the usage of it and removed it from the `I18nOptions` type.